hvplot/converter.py

```diff
@@ -334,6 +334,24 @@ class HoloViewsConverter:
     -------------
     bgcolor (default=None): str
         Background color of the data area of the plot
+    color: str or list, optional
+        Color(s) to use for the plot. Accepts a single color, a list of colors, or
+        a column name from the data to map colors based on values.
+    c: str or list, optional
+        Alias for `color`. Used for specifying colors in scatter plots and other
```

Comment:
Why is the description different than `color`? In other words, is it strictly an alias? What happens if both are provided?

Reply:
It would be kinda redundant to repeat the exact same descriptions both places, no?

It both are provided, `color` takes precedence. I think that can be documented.